src/sagemaker/processing.py

```diff
@@ -739,6 +739,94 @@ def stop(self):
         """Stops the processing job."""
         self.sagemaker_session.stop_processing_job(self.name)
 
+    @staticmethod
+    def prepare_app_specification(container_arguments, container_entrypoint, image_uri):
+        """Prepares a dict that represents a ProcessingJob's AppSpecification
```

Comment:
nit-pep257

Reply:
Fixed in the next revision. (moved the extra information to describe what the function returns.)